packages/size-limit/get-config.js

```diff
@@ -91,6 +96,18 @@ function toName(files, cwd) {
  */
 const dynamicImport = async filePath => (await import(filePath)).default
 
+/**
```

Comment:
Let’s not create TSDoc only for single function. The functions content is pretty self-explanatory.

Reply:
I mean it's just a brief explanation, it's not really huring anybody. Do you still want me to remove it?